zng/docs/spec.md

```diff
@@ -335,6 +346,46 @@ subsequent key for values that have previous keys of equal value.
 It is an error for any such values to appear that contradicts the most
 recent ordering directives.
 
+### 2.1.3 Frame Markers
```

Comment:
This comment probably belongs in section 2.1.2 but I can't comment there: are ordering hints scoped to their frame or global?

Reply:
Good question.  One of the things nagging at me about this effort is that the goal here is to be able to build an external index, but we don't have any real principled reasoning about what goes in-band and what is kept externally.
The idea of the external index is that it only works if the bzng file is sorted and contains frame markers.  If the presence of an index means that the thing that created the index verified that these things are true, then what's the point of adding an additional in-band hint?  If it is just for some hypothetical future consumer, then we don't have enough information to answer your question :/